scripts/versioning-manager.sh

```diff
@@ -269,6 +269,68 @@ elif [ "$VERSIONING_ACTION" == "DELETE_TENANT_PIN" ]; then
     exit 1
   fi
 
+elif [ "$VERSIONING_ACTION" == "CLONE_EXCLUSIVE_RELEASE" ]; then
+  echo "## creating an exclusive cloned release"
+  if [ -z "$EXCLUSIVE_TENANCY" ]; then
+    echo "## no EXCLUSIVE_TENANCY provided or found, exiting"
+    exit 2
+  fi
+  if [ -z "$CLONED_RELEASE_NUMBER" ]; then
+    echo "## no CLONED_RELEASE_NUMBER provided or found, exiting"
+    exit 2
+  fi
+
+  REQUEST_BODY='{
+    "exclusiveTenancy": "'"$EXCLUSIVE_TENANCY"'",
+    "cloneReleaseNumber": "'"$CLONED_RELEASE_NUMBER"'",
```

Comment:
The request body has this parameters  
{
  "releaseNumber": "string",
  "version": "string",
  "exclusiveTenancy": "string",
  "cloneReleaseNumber: "string",
  "environment": "string",
  "releaseDate": "2023-08-18T09:10:10.674Z",
  "endOfLife": "2023-08-18T09:10:10.674Z", - maybe set it in 6 months as a default
  "lts": false, - optional
  "releaseStatus": "PENDING",
  "releaseNotesTitle": "string" - optional
}